src/pages/challenges.tsx

```diff
@@ -29,20 +27,16 @@ const renderCardList = (
 
 const Challenges: FC = () => {
     const [filteredChallenge, setFilteredChallenge] = useState<any>([]);
-    const { data: user } = useGetUserQuery(skipToken);
-    const { response, error, isLoading, callAPI } = useFetch(
-        CHALLENGES_URL,
-        {},
-        false
-    );
+    const { data, isLoading, error } = useGetChallengesQuery();
+    
+    const challengesResponse = data?.challenges;
```

Comment:
have you verfied the eif it is only error or isError key gettting destructured from useGetChallengesQuery()

Reply:
You are right, its isError, I have verified it. will fix,